### DISCUSSION TITLE
DNS Rewrites from GitHub.

### DISCUSSION THREAD

**Discussion by jacktheseb** [AdguardTeam/AdGuardHome#6098] [Ideas]
Hi I want to save my DNS rewrites in an Github file.
The background, I have on multiple sites an Adguard installation. I don't want to add a rewrite on every site so I want to add the rewrite at one place and every site syncs from there.

**Comment by ainar-g** [comment]
Why not just use the [`dnsrewrite` rules](https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#dnsrewrite)?
